bin/build

```diff
@@ -4,13 +4,10 @@ TAG=latest
 
 ARCH=`uname -m`
 
-RPM_BUILD_OPTIONS=${RPM_BUILD_OPTIONS:-""}
-RPM_BUILD_IMAGE=${RPM_BUILD_IMAGE:-"manageiq/rpm_build:$TAG"}
-RPM_PREFIX=${RPM_PREFIX:-"manageiq"}
```

Comment:
I dont't understand removing these as they are used down below in https://github.com/ManageIQ/manageiq-pods/pull/959/files#diff-1fba65931954c73107694f0b7b964148febc5ba35da6812e1e117dfb37436e77R42-R47

Reply:
That being said I never understood building the RPMs as part of this script - always felt like if that's what's wanted the rpms should be built separately and then passed in.